base/common/ReadlineLineReader.cpp

```diff
@@ -107,6 +109,8 @@ ReadlineLineReader::ReadlineLineReader(
         throw std::runtime_error(std::string("Cannot set signal handler for readline: ") + strerror(errno));
 
     rl_variable_bind("completion-ignore-case", "on");
+    // TODO: it doesn't work
+    // history_write_timestamps = 1;
```

Comment:
Looks like a stale comment. If builds are ok, you may remove it.

Reply:
It's left deliberately to indicate that further investigation is required...